### PR TITLE
test(ivy): improve `LViewDebug` information

### PR DESCRIPTION
**Pull request by mhevery**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [X] Other... Please describe: Improved Debugging experience

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: packages/core/src/render3/util/view_utils.ts [hunk 1]
```diff
@@ -90,6 +90,7 @@ export function getNativeByIndex(index: number, lView: LView): RNode {
  * @param lView
  */
 export function getNativeByTNode(tNode: TNode, lView: LView): RNode {
+  // TODO(misko): rename to getRNodeFromLView
```
Comment (pkozlowski-opensource) [inline packages/core/src/render3/util/view_utils.ts]: not sure about this rename - note that we've got another helper function (`getNativeByIndex`) so if you rename `getNativeByTNode`-> `getRNodeFromLView ` the other one should be renamed to `getRnodeByIndexFromLView`?

Generally speaking, I'm not sure if adding `FromLView` is needed as we've got this information in the argument names / types already.

tl;dr; not sure about this rename and would remove this TODO
